### PR TITLE
validation web-ui

### PR DESCRIPTION
**Pull request by ObiWahn**

### CODE REVIEW

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/validationView.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  'use strict';
+
+  window.ValidationView = Backbone.View.extend({
+    el: '#content',
+    readOnly: false,
+    template: templateEngine.createTemplate('validationView.ejs'),
+
+    initialize: function (options) {
+      this.collectionName = options.collectionName;
+      this.model = this.collection;
```
[the remark this answers is not in the document]
Comment (ObiWahn) [reply js/apps/system/_admin/aardvark/APP/frontend/js/views/validationView.js:15]: Yes - I use the model to get the collection properties and change them.

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/validationView.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+      this.resize();
+      var options = {
+        onChange: function () {
+          //self.jsonContentChanged();
+        },
+        onModeChange: function (newMode) {
+          //self.storeMode(newMode);
+        },
+        search: true,
+        mode: 'code',
```
[the remark this answers is not in the document]
Comment (ObiWahn) [reply js/apps/system/_admin/aardvark/APP/frontend/js/views/validationView.js:47]: yes

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/validationView.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+          this.model.changeValidation(newprops, (err, data) => {
+            if(err) {
+              arangoHelper.arangoError('Error', 'Could not update validation for' + this.model.get('name') + '.');
+            } else {
+              arangoHelper.arangoNotification('Saved validaiton for collection ' + this.model.get('name') + '.');
+            }
+          });
+
+        } // if error
```
[the remark this answers is not in the document]
Comment (ObiWahn) [reply js/apps/system/_admin/aardvark/APP/frontend/js/views/validationView.js:98]: This is intentional.

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/validationView.js [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+      this.editor = new JSONEditor(container, options);
+    },
+
+    getValidationProperties: function () {
+      var propCB = (error, data) => {
+        if (error) {
+          window.arangoHelper.arangoError("Error", "Could not get collection properties");
+        } else {
+          console.log(data);
```
Comment (hkernbach) [inline js/apps/system/_admin/aardvark/APP/frontend/js/views/validationView.js:57]: still console log

**Pull request merged**